<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConductor</class>
 <widget class="QMainWindow" name="frmConductor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conductor Table</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/FormIcons/resources.qrc">
    <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/formconductor.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/formconductor.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>86</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>21</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Project</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPro">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>18</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>21</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>51</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Substation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>51</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Feeder</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSub">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>48</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPBS">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>18</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>48</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>12</y>
       <width>62</width>
       <height>62</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/plugins/ElectricSystems/Resources/FormIcons/basicnreca.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdConEdit">
     <property name="geometry">
      <rect>
       <x>685</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Edit Conductor Table</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/tableedit.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/tableedit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdConSave">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Save Conductor Table</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/tablesave.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/tablesave.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdHelp">
     <property name="geometry">
      <rect>
       <x>795</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Help</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basichelp.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basichelp.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdClose">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Close</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basicexit.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basicexit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="dgvTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>107</y>
      <width>921</width>
      <height>411</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/FormIcons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
